app/src/main/java/woowacourse/shopping/data/goods/repository/GoodsRemoteDataSourceImpl.kt

```
@@ -0,0 +1,121 @@
+                    override fun onFailure(
+                        call: Call<GoodsResponse>,
+                        t: Throwable,
+                    ) {
+                        onFailure(t)
+                    }
+                },
+            )
+    }
+
+    override fun fetchGoodsSize(onComplete: (Int) -> Unit) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)